packages/playwright-crawler/src/internals/adaptive-playwright-crawler.ts

```diff
@@ -317,12 +325,12 @@ export class AdaptivePlaywrightCrawler extends PlaywrightCrawler {
                                         );
                                     },
                                     () =>
-                                        this.adaptiveRequestHandler({
+                                        this.adaptiveRequestHandler!({
                                             request: crawlingContext.request,
                                             log: crawlingContext.log,
                                             querySelector: async (selector, timeoutMs) => {
                                                 const locator = playwrightContext.page.locator(selector).first();
-                                                await locator.waitFor({ timeout: timeoutMs });
+                                                await locator.waitFor({ timeout: timeoutMs, state: 'attached' });
```

Comment:
@janbuchar this helps with the timeout issue when trying to query for hidden elements like `html title`, i hope it won't break anything (but hey, its experimental anyway)